### PR TITLE
Simple paper

### PR DESCRIPTION
**Pull request by PalumboN**
Fixes to generate a first version of SimpleDruid.

- Refactoring tests to run on SimpleDruid too
- Moving and cleaning general tests
- Fix `LoadStackValue` and `LoadArgument` instructions generation for n > 1

### VM chenges: https://github.com/pharo-project/pharo-vm/pull/869

### CODE REVIEW

**Comment by PalumboN** [comment]
Running the tests in SimpleDruid makes the CI timeouts:
```
Error: The action 'Load Image and Run Tests' has timed out after 30 minutes.
```

File: Druid/DRBytecodeIRGenerator.class.st [hunk 1]
```diff
@@ -360,19 +360,21 @@ DRBytecodeIRGenerator >> interpretSendWith: aRBMessageNode specialSelector: isSp
...
+				marshall addBefore:
+					(controlFlowGraph instructionFactory flushStackExceptTop: argumentCount simpleConstantFold + 1).
+				argumentCount simpleConstantFold + 1 "Receiver" timesRepeat: [
+					self
+						pop: 1
+						unspilled: argumentCount simpleConstantFold <= self numRegArgs ] ]
+			ifFalse: [
+				marshall addBefore: controlFlowGraph instructionFactory flushStack.
+				marshall doPop: true ].
+		^ self push: DRPhysicalGeneralPurposeRegister receiverResultReg ]
```
[the remark this answers is not in the document]
Comment (PalumboN) [reply Druid/DRBytecodeIRGenerator.class.st:377]: https://github.com/Alamvic/druid/issues/159

File: Druid/DRCogitSimpleStackGenerator.class.st [hunk 2]
```diff
@@ -157,6 +157,12 @@ DRCogitSimpleStackGenerator >> visitClosureCreation: aDRClosureCreation [
 	copy acceptVisitor: self
 ]
 
+{ #category : #visiting }
+DRCogitSimpleStackGenerator >> visitCogitSendMarshall: aDRCogitSendMarshall [ 
+
+	"Avoid marshalling in SimpleStack"
+]
+
```
[the remark this answers is not in the document]
Comment (PalumboN) [reply Druid/DRCogitSimpleStackGenerator.class.st:165]: https://github.com/Alamvic/druid/issues/160

**Comment by PalumboN** [comment]
I extended the timeout and is working (the half).
Comments where moved to issues.

I merge to continue with the next PR

**Comment by guillep** [comment]
Yes!

**Pull request merged**